test/homework_01/test_task_03.py

```diff
@@ -1,9 +1,15 @@
 from homework_01.task_03 import find_maximum_and_minimum
 
 
-def test_1_5():
-    assert find_maximum_and_minimum("./test_task03_files/file0.txt") == (1, 5)
+def test_12345():
+    assert find_maximum_and_minimum("test/homework_01/test_task03_files/file0.txt") == (
+        1,
+        5,
+    )
 
 
 def test_0():
```

Comment:
This test gives me zero information because it is not obvious that it is not a test number but some name schema.   
Lets make it easy visible: `test_numbers_0_return_0_and_0`